Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.cpp

```diff
@@ -960,6 +960,16 @@ namespace AzToolsFramework
         return m_domPath;
     }
 
+    bool DPERowWidget::HasChildRows() const
+    {
+        return !m_domOrderedChildren.empty();
+    }
+
+    int DPERowWidget::GetLevel() const
+    {
+        return aznumeric_cast<int>(m_domPath.size() - 1);
```

Comment:
nit: since size_t is unsigned, it would make more sense to cast before subtracting:
```
return aznumeric_cast<int>(m_domPath.size()) - 1;
```

Reply:
Done! This function gets called many times when loading a DPE that has many widgets so better to fix it.